rfc/text/advanced/ap_50_payload_transparency.md

```diff
@@ -0,0 +1,251 @@
+## Payload Transparency
```

Comment:
"Transparent" means that the contents of the payload is visible. The terminology you need to use is "opaque payload" or "payload opacity".

https://www.merriam-webster.com/dictionary/transparent
https://www.merriam-webster.com/dictionary/opaque
https://www.merriam-webster.com/dictionary/opacity
https://english.stackexchange.com/questions/87177/difference-between-opacity-and-opaqueness